tests/testcases/core/helpers/EEH_File_Test.php

```diff
@@ -30,10 +30,13 @@ function setUp()
 
     function tearDown()
     {
+        // restore to using the normal WP filesystem
         global $wp_filesystem;
+        $wp_filesystem = null;
+        unset($wp_filesystem);
```

Comment:
Can we just use `unset($GLOBALS['wp_filesystem'])` here ?

Reply:
hmmm, lemme try that...